docs/how-to-guides/run-and-evaluate-a-flow/index.md

```diff
@@ -155,7 +155,7 @@ pf run visualize -n "my_first_run,my_first_eval_run"
 
 A web browser will be opened to show the visualization result.
 
-![q_0](../../media/how-to-guides/visualize_run.png)
+![q_0](../../media/how-to-guides/run_visualize.png)
```

Comment:
delete the old png ?

Reply:
also change the place reference it

Reply:
There are two visualized pictures, do you need to change another one?

Reply:
Good catch, update.